### ISSUE TITLE
Drop an item from one grid to another

### ISSUE THREAD

**Issue by yamsellem** [ducksboard/gridster.js#115]
Is there a way to drag items across multiple gridster?

**Comment by lukju** [comment]
+1 
i would like to create something like the Win8 start screen - which also has multiple containers and dragging between containers is possible there. 
so, drag'n'drop between different gridsters would be a great feature...

**Comment by rochapablo** [comment]
@yamsellem, @lukju have guys you found something?

**Comment by lukju** [comment]
Not really but I've solved my requirement as follow:
- there is an Edit Mode in my application. Moving widgets around is only possible while in edit mode.
- when in edit mode, there are two additional icons visible, "move to next container" and "move to prev container". user has to click on these icon commands in order to change the widget's container. I'm using Gridster's event of removing and adding widgets.

So, in short: I haven't changed Gridster, i've built it around it. User Experience would clearly be better when there would be a drag'n'drop way of changing the container. But it works for us and it is pretty easy to work with.

Good luck.
Lukju
